app/lib/getOpeningTimesMessage.js

```diff
@@ -0,0 +1,38 @@
+function getOpeningHoursMessage(status) {
+  if (status.isOpen) {
+    const closedNext = status.until;
+    const closedTime = closedNext.format('h:mm a');
+    const closedDay = closedNext.calendar(status.moment, {
+      sameDay: '[today]',
+      nextDay: '[tomorrow]',
+      nextWeek: 'dddd',
+      lastDay: '[yesterday]',
+      lastWeek: '[last] dddd',
+      sameElse: 'DD/MM/YYYY',
+    });
+    return (
+      ((closedDay === 'tomorrow' && closedTime === '12:00 am')
```

Comment:
Could this be extracted out into a function to help explain what it is doing? Something like `isClosedTodayAndTomorrow()`